azurelinuxagent/common/utils/shellutil.py

```diff
@@ -111,7 +111,10 @@ def run_get_output(cmd, chk_err=True, log_cmd=True, expected_errors=[]):
 
 
 def _encode_command_output(output):
-    return ustr(output, encoding='utf-8', errors="backslashreplace")
+    if output:
```

Comment:
Removing this since I won't be using `run_command` for running shell commands from the log collector utility. I needed this because `stdout` or `stderr` were `None` in some of my use cases.